src/coll.rs

```diff
@@ -183,6 +183,16 @@ impl<T> Collection<T> {
         }
     }
 
+    pub(crate) fn clone_unconcerned(&self) -> Self {
```

Comment:
Is this necessary? IIRC these options are propagated from the client when a db/coll is instantiated and then resolved with the `resolve_options!` macro, so I think they will only get applied if they're explicitly folded into the options provided to a method (which we're not doing in these helpers)

Reply:
It's needed for `list_search_indexes`, which is implemented as an `aggregate` call.